### DISCUSSION TITLE
Showing status on all nodes in the deployment in SOC

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#1862] [Unsupported Versions] [answered]
It would be nice if the status of all the nodes was showed in the SOC, so it was easier to see if it was any problems with the nodes
in the sensors tab, one group for sensor, search, fleet,heavy, master. if they are present in the deployment

Is something like this planned?

**Comment by weslambert** [comment, marked as answer]
We currently can see the status using Salt via CLI and in Grafana.  We may consider adding something to SOC for basic status information, but nothing is planned at this time, to my knowledge.
